client-spark/spark-3/src/main/java/org/apache/spark/shuffle/celeborn/SortBasedShuffleWriter.java

```diff
@@ -191,8 +190,13 @@ public void write(scala.collection.Iterator<Product2<K, V>> records) throws IOEx
 
   @VisibleForTesting
   boolean canUseFastWrite() {
```

Comment:
@waitinfuture do you have benchmark results, how fast can this achieve?

Reply:
> @waitinfuture do you have benchmark results, how fast can this achieve?

I think it can make a big difference because UnsafeRow's serializer just copy memory from UnsafeRow. Though I haven't benchmark it. We can do a benchmark if necessary.